buildlib/dockers/ubuntu-release.Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN apt-get update && \
         libnvidia-compute-${NV_DRIVER_VERSION} \
         make \
         maven \
+        pkg-config \
```

Comment:
Why do we need this?

Reply:
Required for compilation. 
Ubuntu22 base image doesn't have it installed, unlike all previous Ubuntu image versions.